atypicals_wrangler.py

```diff
@@ -0,0 +1,128 @@
+import json
+import boto3
+import traceback
+import pandas as pd
+import os
+import random
+
+
+def _get_traceback(exception):
+    """
+    Given an exception, returns the traceback as a string.
+    :param exception: Exception object
+    :return: string
+    """
+
+    return ''.join(
+        traceback.format_exception(
+            etype=type(exception), value=exception, tb=exception.__traceback__
+        )
+    )
+
+
+def lambda_handler(event, context):
+    # Set up clients
+    # s3 = boto3.resource('s3')
+    sqs = boto3.client('sqs')
+    lambda_client = boto3.client('lambda')
+    sns = boto3.client('sns')
+
+    # Sqss
+    queue_url = os.environ['queue_url']
+    sqs_messageid_name = os.environ['sqs_messageid_name']
+
+    # Sns
+    arn = os.environ['arn']
+    checkpoint = os.environ['checkpoint']
+
+    atypical_columns = os.environ['atypical_columns']
+
+    method_name = os.environ['method_name']
+
+    # bucket_name = os.environ['bucket_name']
+    input_data = os.environ['input_data']
+
+    try:
+
+        #  Reads in Data from SQS Queue
+        response = sqs.receive_message(QueueUrl=queue_url)
+        message = response['Messages'][0]
+        message_json = json.loads(message['Body'])
+
+        # Reads in data from S3 Bucket into a JSON file
+
+        # content_object = s3.Object(bucket_name, input_data)
```

Comment:
If not being used then best to remove.

Reply:
We left in the stuff about getting from a bucket rather than the SQS queue because we weren't sure if we'd need to return to retrieving from a bucket if the data became to big. Has this issue been completely resolved now?

Reply:
Taht's a fair point. Still think this is ongoing and will become a bigger issue when we try to pass larger data through SQS, so maybe it is best to leave it in for now.